src/pages/players/Players.tsx

```diff
@@ -58,7 +58,7 @@ class Players extends React.Component<{}, { players: Player[] }> {
                         <tbody>
                         {
                             this.state.players.map((player) => (
-                                <tr className="players-row">
+                                <tr className="players-row" key={player.id}>
```

Comment:
Thanks for adding this! :) Will be nice not to see that console warning about the key anymore! :D